timepiece/entries/forms.py

```diff
@@ -4,15 +4,20 @@
 from django import forms
 from django.db.models import Q
 
+from selectable import forms as selectable
 
 from timepiece import utils
 from timepiece.crm.models import Project, ProjectRelationship
 from timepiece.entries.models import Entry, Location, ProjectHours
+from timepiece.entries.lookups import ActivityLookup
 from timepiece.forms import (
     INPUT_FORMATS, TimepieceSplitDateTimeField, TimepieceDateInput)
 
 
 class ClockInForm(forms.ModelForm):
+    activity = selectable.AutoComboboxSelectWidget(
```

Comment:
With the widget definition in the `Meta` blow, you can remove this field definition entirely.